lesson_15/lesson_15.py

```diff
@@ -0,0 +1,40 @@
+# Створіть клас геометричної фігури "Ромб". Клас повинен мати наступні атрибути:
+#
+# сторона_а (довжина сторони a).
+# кут_а (кут між сторонами a і b).
+# кут_б (суміжний з кутом кут_а).
+# Необхідно реалізувати наступні вимоги:
+#
+# Значення сторони сторона_а повинно бути більше 0.
+# Кути кут_а та кут_б повинні задовольняти умову: кут_а + кут_б = 180
+# Протилежні кути ромба завжди рівні, тому при заданому значенні кут_а, значення кут_б обчислюється автоматично.
+# Для встановлення значень атрибутів використовуйте метод setattr.
+
+
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        setattr(self, 'side_a', side_a)
```

Comment:
This attributes is better to assign proper with self access. For ex self.side_a = side_a. Additional methods with setattr  is able to only use assigning via method only. Thanks